randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3312,7 +3321,7 @@ Extra - asset_id: collision_camera_042
   * Layers: default
   * Open Passage to Lava Button West/Dock to Thermal Device Room North (Mid)
   > Dock to Dairon Transport Access
-      Spider Magnet or Infinite Bomb Jump (Beginner) or Use Spin Boost
+      Spider Magnet or Speed Booster or Infinite Bomb Jump (Beginner) or Use Spin Boost
```

Comment:
Change the IBJ (Beginner) to Simple IBJ. Rather, just make this trivial cause wall jumps.